registration/tests/test_views.py

```diff
@@ -598,15 +598,15 @@ def test_preserve_empty_privacy_url(self):
         """
         self._register(self.form_data)
         application = BetaTestApplication.objects.get()
-        application.privacy_policy_url = None
+        application.privacy_policy_url = ''
         application.save()
         application.refresh_from_db()
-        assert application.privacy_policy_url is None
+        assert application.privacy_policy_url == ''
```

Comment:
Change this to self.assertEqual (ref https://docs.python.org/3/library/unittest.html#assert-methods).

Reply:
The use of `assert` rather than `self.assertEqual` in this instance is intentional: we're asserting that the test setup has been performed correctly, rather than making assertions about the behaviour of the code under test. Happy to switch it, but I've found this distinction useful when things don't go according to plan.

Reply:
We don't use `assert` like this in any other tests, so it's better for us to use the same pattern. Please change the assert to `self.assertEqual`.